packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -23,23 +23,23 @@
   transition-duration: 100ms;
   transition-timing-function: linear;
   border: 1px solid transparent;
-  background-color: $color-primary-base;
+  background-color: var(--ColorBackgroundPrimary);
```

Comment:
I'd prefer kebab case CSS vars unless there's an issue preventing that. It'll match other CSS values and feels easier to recognize token demarcation.

ex. `--color-background-primary` vs `--ColorBackgroundPrimary`

Reply:
I suppose I don't feel strongIy here but I went this route because this was the intended format in the writeup from @arnoldsandoval-okta about token naming. Keep in mind that these values are provided from the JS side too so using kebab case would require an uglier explicit string object property key like this:

```
{
  'color-background-primary': ...
}
```

That seems less than ideal to me

Reply:
After @jeffbelser-okta's walkthrough today, I understand the syntax choice. Good with the current setup!